File: Backend/src/EventScheduler/EventScheduler.Domain/Event.cs
using System;
using System.Collections.Generic;
using EventScheduler.Domain;

namespace EventScheduler.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Place { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int PeopleQuantity { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lot> Lotes { get; set; }
        public IEnumerable<SocialNetwork> RedesSociais { get; set; }
        public IEnumerable<SpeakerEvent> PalestrantesEventos { get; set; }
    }
}

File: Frontend/EventScheduler/src/app/models/Event.ts
import { Lot } from "./Lot";
import { SocialNetwork } from "./SocialNetwork";
import { SpeakerEvent } from "./SpeakerEvent";

export interface Event {
     id : number; 
     place : string; 
     eventDate? : Date; 
     theme : string;  
     peopleQuantity : number; 
     imageURL : string; 
     phone : string; 
     email :string; 
     lots? : Lot[] ; 
     socialNetworks? : SocialNetwork[];
     speakersEvents? : SpeakerEvent[]; 
}
